package.json

```diff
@@ -12,10 +12,12 @@
     "devnet": "ENABLE_MAINNET_FORKING=true NO_RESET=true npx hardhat node",
     "console:rinkeby": "npx hardhat console --network rinkeby",
     "console:ropsten": "npx hardhat console --network ropsten",
-    "clean": "rm -rf build && rm -rf out",
-    "test": "forge test",
+    "clean": "forge clean && rm -rf artifacts",
+    "test": "forge test --no-match-contract IntegrationTest",
+    "test:integration": "forge test --fork-url https://eth-mainnet.alchemyapi.io/v2/$MAINNET_ALCHEMY_API_KEY --fork-block-number `cat ./contracts/test/integration/_block.txt` --match-contract IntegrationTest",
```

Comment:
we should really change this and just leverage an env var instead of catting a file.

Reply:
Agreed, we absolutely shouldn't be using operating-specific commands here either

Reply:
Pushed changes and latest feedback regarding integration tests to the PR: https://github.com/fei-protocol/fei-protocol-core/pull/565 . Forked off from that as I need it here